app/src/main/java/tech/ula/model/repositories/AssetRepository.kt

```diff
@@ -24,7 +24,8 @@ data class DownloadMetadata(
 class AssetRepository(
     private val applicationFilesDirPath: String,
     private val assetPreferences: AssetPreferences,
-    private val githubApiClient: GithubApiClient = GithubApiClient(),
+    private val ulaFiles: UlaFiles,
+    private val githubApiClient: GithubApiClient = GithubApiClient(ulaFiles),
```

Comment:
I think the initialization of `GithubApiClient` should probably be handled by whatever is constructing the `AssetRepository`, in order to match code style in the rest of the app. That is to say, we shouldn't use a default parameter here.

Reply:
True.  This has been made to follow the style.